projects/storefrontapp/karma.conf.js

```diff
@@ -11,11 +11,18 @@ module.exports = function (config) {
       require('karma-chrome-launcher'),
       require('karma-jasmine-html-reporter'),
       require('@angular-devkit/build-angular/plugins/karma'),
+      require('karma-junit-reporter'),
```

Comment:
We don't need reports on this one since we don't ship it

Reply:
very good catch, and now I'm wondering why do we have this file and the 'test' section in the angular.json for storefrontapp